Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs

```diff
@@ -17,6 +17,12 @@ public class DefaultPointerMediator : IMixedRealityPointerMediator
         protected readonly Dictionary<IMixedRealityInputSource, HashSet<IMixedRealityPointer>> pointerByInputSourceParent = new Dictionary<IMixedRealityInputSource, HashSet<IMixedRealityPointer>>();
 
         private IPointerPreferences pointerPreferences;
+
+        public DefaultPointerMediator()
```

Comment:
nit: while you're here, can you add a doccommunet to the class

Reply:
I don't know this class well enough to describe it in a sentence.  I just hit an issue when getting latest mrtk_development and made a change to get the old behavior back.  If you tell me what to write here, I'll be happy to add it while I'm here.

Reply:
@julenka to confirm but something like
"The default implementation for pointer mediation in MRTK which is responsible for determining which pointers are active based on the state of all pointers. For example, one of the key things this class does is disable far pointers when a near pointer is close to an object."

Reply:
Troy's comment is great, thanks Troy!

Reply:
done